src/web/app/components/question-responses/grouped-responses/grouped-responses.component.html

```diff
@@ -2,10 +2,14 @@
     *ngIf="responses.length && responses[0].allResponses.length">
   <div class="col-md-3">
     <div>
-      {{ isGrq ? 'To' : 'From' }}: {{ responses[0].allResponses[0][isGrq ? 'recipient' : 'giver']}} {{ teamInfo[isGrq ? 'recipient' : 'giver'] }}
+      <tm-view-photo-popover [photoUrl]="userToPhotoUrl[topName]" (loadPhotoEvent)="loadPhotoEvent.emit(topName)">
+        {{ isGrq ? 'To' : 'From' }}: {{ topName }} {{ teamInfo[isGrq ? 'recipient' : 'giver'] }}
```

Comment:
I don't think `topName` is a good name. Actually the original expression looks better

Reply:
Okay will revert to original expr